core/test/time_test.jl

```diff
@@ -29,5 +29,5 @@
     t = Ribasim.seconds_since.(flow_1_to_2.time, model.config.starttime)
     flow_expected = @. 1 + sin(0.5 * π * (t - t[1]) / (t[end] - t[1]))^2
     # some difference is expected since the modeled flow is for the period up to t
-    @test isapprox(flow_added_1, flow_expected, rtol = 0.005)
+    @test isapprox(flow_added_1, flow_expected, rtol = 0.01)
```

Comment:
Since the saved flows are no longer instantaneous, they should not be used for comparing against exact solutions any further

Reply:
Yeah that's a bit unfortunate. I guess as long as `saveat` is short enough and the tolerance large enough we can keep using it?

Reply:
Turns out we can, I wrote this before I noticed that initially my averaged flows were all wrong (: